src/vecgf2.c

```diff
@@ -1463,21 +1463,22 @@ Obj NewGF2Vec (
           SetTypeDatObj(res, TYPE_LIST_GF2VEC_IMM);
         return res;
     }
+
+    if (!IS_LIST(list)) {
+        ErrorMayQuit("COPY_GF2VEC: argument must be a list of compressed GF2 vectors", 0L, 0L);
```

Comment:
Should this rather be `argument must be a list of GF2 elements`?
- it should be elements not vectors
- If I read the code correctly, it can also handle GF2 elements over some arbitrarily huge field. Or are elements GFq, q = 2^r _always_ compressed?

Reply:
You are right about the first one (I blame this on copy&paste). I don't understand what you mean with the second one -- what is a "GF2 element over some arbitrarily huge field" ? There are exactly two elements of GF2, period (albeit possibly in different reps).

Reply:
Ah, I was confused by the term _compressed_ wrt different representations.